Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: zabbix
-version: 0.2.1
+version: 0.2.2
```

Comment:
Hi @anxolerd!

In **develop** branch the version of chart is 0.3.0. Can you change here to 0.3.1 or major?